consensus/istanbul/backend/backend.go

```diff
@@ -251,6 +255,27 @@ func (sb *Backend) Commit(proposal istanbul.Proposal, bitmap *big.Int, seals []b
 	if err != nil {
 		return err
 	}
+
+	parent := sb.chain.GetHeader(h.ParentHash, h.Number.Uint64()-1)
+	// If the parent was not the genesis block then try to use the parent's
+	// extras as the seal for this block
+	if parent != nil {
+		parentExtras, err := types.ExtractIstanbulExtra(parent)
+		if err != nil {
+			return err
+		}
+
+		block = block.WithParentSeal(
```

Comment:
Is `Commit` the right place to do this? I would have thought we would do this in `Prepare`, after we sleep, to give us the most time possible to collect signatures